js/app.js

```diff
@@ -7,9 +7,13 @@ let score = 0;
 //console.log('Hello World');
 // console log finally responded with correct messaging. //
 
+function username(){
 let username = prompt('What is your name?');
 alert('Hello ' + username) + '!';
+}
+username();
```

Comment:
Try to call your functions all in one spot. The standard location for this is at the end of your document. This really helps with organizing your code and debugging.